src/Storages/ColumnsDescription.h

```diff
@@ -153,6 +153,9 @@ class ColumnsDescription
     String toString() const;
     static ColumnsDescription parse(const String & str);
 
+    /// `column1` type1, `column2` type2, ...
+    String toExpressionList() const;
```

Comment:
We already have similar method, see `DatabaseOnDisk::getCreateQueryFromStorage`

Reply:
Thanks, will check it